demos/http/http_demo_mutual_auth/demo_config.h

```diff
@@ -83,21 +85,14 @@
     #define ROOT_CA_CERT_PATH    "certificates/AmazonRootCA1.crt"
```

Comment:
Why did we change the definitions of the client cert/key paths here but not the root CA cert?

Reply:
The Amazon Root CA is automatically downloaded by CMake, and so this certificate won't change anytime soon for IoT Core.